boot/seal.go

```diff
@@ -141,20 +139,43 @@ func fallbackKeySealRequests(key, saveKey secboot.EncryptionKey) []secboot.SealK
 	}
 }
 
+type fdeSetupHookResult struct {
+	EncryptedKey []byte `json:"encrypted-key"`
+	Handle       []byte `json:"handle"`
+}
+
 func sealKeyToModeenvUsingFDESetupHook(key, saveKey secboot.EncryptionKey, model *asserts.Model, modeenv *Modeenv) error {
```

Comment:
should we pass a different value to stampSealedKeys for v2 hooks? or is not worth it?

Reply:
So far the code tries to keep all filenames the same and do pure content detection. So I'm inclined to keep it as is or switch away from pure content detection.